syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -661,7 +661,7 @@ def attr(self, attr_name):
 
         return attr_val
 
-    def clone(self):
+    def clone(self, *args, **kwargs):
```

Comment:
We need to make sure that we take into account the arguments sent to clone

Reply:
Ahhh..yes :( I should

Reply:
This is needed because of [this] (https://github.com/pytorch/pytorch/blame/master/torch/nn/parameter.py#L32) - it was added 3 months ago.